ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Storage/PerasCertDB/Impl.hs

```diff
@@ -215,7 +217,9 @@ implGetCertSnapshot PerasCertDbEnv{pcdbVolatileState} =
 implGarbageCollect ::
   forall m blk.
   (IOLike m, StandardHash blk) =>
-  PerasCertDbEnv m blk -> SlotNo -> m ()
+  PerasCertDbEnv m blk ->
+  SlotNo ->
+  m ()
```

Comment:
This signature didn't change, so I'm not sure why this change crept in.

If fourmolu running on some lenient/not-strict mode?